### DISCUSSION TITLE
devtools doesn`t work in FireFox

### DISCUSSION THREAD

**Discussion by EugeneKamikaZe** [pmndrs/zustand#1597] [General]
DevTools work good in Chrome, but doesn`t work in FireFox

**Comment by dai-shi** [comment]
Can anyone help how we can fix it?

**Comment by voinik** [comment]
Ran into exactly this just now. I uninstalled the extension and installed it back again on Firefox and now it works!

**Comment by AndreiTS** [comment]
It only works if you have the devtools tab open while loading the page
